Middlewares/ValidationFilterAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.Filters;
+using Newtonsoft.Json;
+using Sample.Constant;
+using Sample.Dtos;
+using Sample.Exceptions;
+
+namespace Sample.Middlewares;
+
+public class ValidationFilterAttribute : IActionFilter
+{
+    public void OnActionExecuting(ActionExecutingContext context)
+    {
+        if (!context.ModelState.IsValid)
```

Comment:
chỗ này mình
 ``
```suggestion
        if (!context.ModelState.IsValid)
```
if (context.ModelState.IsValid)
        {
            return;
        }

        var errors = context.ModelState.Select(error => new DataFailDto
        {
            Field = error.Key,
            Error = error.Value?.Errors.Select(modelError => modelError.ErrorMessage).ToList()
        }).ToList();

Reply:
cho nó clean, nghĩa là nó valid thì cho lượn luôn